libtcmu_log.c

```diff
@@ -275,12 +276,26 @@ static int append_output(log_output_fn_t output_fn, log_close_fn_t close_fn, voi
 	output.priority = pri;
 	output.dest = dest;
 	output.name = ndup;
+	output.enabled = true;
 
 	darray_append(logbuf->outputs, output);
 
 	return 0;
 }
 
+static void log_output_disable(const tcmu_log_destination dest)
+{
+	struct log_output *output;
+
+	/* This will just find the first one and disable it. */
+	darray_foreach(output, logbuf->outputs) {
+		if (output->dest == dest) {
```

Comment:
Do we need to also check for enabled == true. This would work:

1. update 1.
2. something does tcmu_log()
3. update 2.

If there were 2 updates back to back

1. update 1.
2. update 2.
3. something does tcmu_log()

then this would only disable the first one twice and the later delayed removal in log_output would only remove the original.